src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -33,7 +33,13 @@ export class CourseMergeOfferingCollectionEntity extends Entity {
 	}
 
 	pageSize() {
-		return this._pagingInfo()?.pageSize;
+		const pageSize = this._pagingInfo()?.pageSize;
+		const totalCount = this.totalCount() ?? 0;
+		const courseMergeOfferingsLength = this.courseMergeOfferings()?.length ?? 0;
+		if (totalCount < courseMergeOfferingsLength + (pageSize ?? 0)) {
+			return totalCount - courseMergeOfferingsLength;
+		}
+		return pageSize;
```

Comment:
Maybe I'm misunderstanding what this is supposed to do, but I'm assuming we want to return the length of the current page?
With the following:
```
pageSize = 20
totalCount = 5
couseMergeOfferingsLength = 5
```
Then `totalCount < courseMergeOfferingsLength + (pageSize ?? 0)` would be `true` so we'd return `0`.
Should this not just be returning `courseMergeOfferingsLength`?

Reply:
We want to return the length of the remaining items. Without this change, as mentioned in the description of the PR, if you have 40 of 50 items loaded, the `pageSize` will still be 20 and the Load More component will say `Load 20 More` even though the 40 of 50 items suggests that there are only 10 left.
To the "Then totalCount < courseMergeOfferingsLength + (pageSize ?? 0) would be true so we'd return 0.", returning 0 is okay. If there are no more items, we shouldn't have gotten a `next` href and the load more button shouldn't show up anymore.

I can add a comment to describe the goal of this.

Reply:
Actually, I'm going to move this to a separate function as we should have a way to return just the `pageSize`.

Reply:
Yea I thought it was returning current page size, but you wanted next page size which is what confused me.